routes/api-routes.js

```diff
@@ -192,12 +192,6 @@ app.get("/api/group_users/:id", function(req, res) {
   })
 });
 
-        // console.log(resultList);
```

Comment:
These were the 5 lines of code that I believe Maria was talking about, needing to be deleted.